src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/RecommendedActionTests.ps1

```diff
@@ -0,0 +1,227 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+
+# ============================================================
+# A NOTE ON (LACK OF) TEST SETUP FOR RECOMMENDED ACTIONS TESTS
+# ============================================================
+# Recommended actions are generated based on database workload and new database won't get any recommended actions.
+# It's hard to generate mock recommendations on a new server, database or elastic pool from this code as well, as the process is Microsoft Internal.
+# For now, Tests have to use recorded responses on some specific database that already has recommended actions.
```

Comment:
Note:  Contributors have to be able to run and re-record these tests.  Please spell out how such a database would be created, if it is diffuclt to do so within the tset.  How is the next contributor going to rerecord these tests?